vcd/resource_vcd_vapp.go

```diff
@@ -395,14 +376,12 @@ func resourceVcdVAppUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 
 		if ovf, ok := d.GetOk("ovf"); ok {
-			err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-				task, err := vapp.SetOvf(convertToStringMap(ovf.(map[string]interface{})))
+			task, err := vapp.SetOvf(convertToStringMap(ovf.(map[string]interface{})))
 
-				if err != nil {
-					return resource.RetryableError(fmt.Errorf("Error set ovf: %#v", err))
-				}
-				return resource.RetryableError(task.WaitTaskCompletion())
-			})
+			if err != nil {
+				return fmt.Errorf("error set ovf: %#v", err)
```

Comment:
"error set ovf" -> "error setting the ovf"

Reply:
Fixed